src/rover_control/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(rover_control rover_control.cpp rover_control.hpp)
+
+find_package(Boost REQUIRED)
+target_include_directories(subsystem PUBLIC ${Boost_INCLUDE_DIR} rover_system_messages)
+target_link_libraries(subsystem PUBLIC network rover_system_messages)
+
+target_compile_features(subsystem PRIVATE cxx_std_17)
```

Comment:
Change `add_executable` to `add_library`.

Also, in the other lines, change the target (first argument) from `subsystem` to `rover_control`. You can also remove the Boost references (find package + include dir)

In the root CMakeLists.txt, add `add_subdirectory(src/rover_control)` between the add subdirectory for network and the base station, but outside of the `NOT NO_GFX` if block